pkgs/misc/qtscrcpy/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+, copyDesktopItems
+, android-tools
+, makeDesktopItem
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qtscrcpy";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "barry-ran";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-2d7AIUra7Uc/N0Ako8JYo07GUTPNgorxl2hmeDJOGPU=";
+  };
+
+  nativeBuildInputs = [ cmake libsForQt5.wrapQtAppsHook copyDesktopItems ];
+  buildInputs = with libsForQt5.qt5 ;[ qttools.dev qtx11extras.dev ];
```

Comment:
```suggestion
  buildInputs = with libsForQt5.qt5; [ qttools.dev qtx11extras.dev ];
```

Reply:
Do you need to explicitly define that you want to use `dev` output? Did you try without?

Reply:
I use the [nix-index](https://github.com/bennofs/nix-index) to find required headers in which packages, I will try without dev.

Reply:
[nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) didn't help me check out the semicolon...